hal/sparcv8leon3/_init.S

```diff
@@ -18,11 +18,118 @@
 #include "config.h"
 #include "../cpu.h"
 
+.extern _plo_load_addr
+.extern _plo_size
+
 .section ".init", "ax"
 .align 4
 .global _init
 .type _init, #function
 _init:
+	/* get current PC */
+	call . + 8
```

Comment:
Won't this cause us to jump to SRAM (regardless if it was initialized and if plo has been copied there)? `.` is provided by the compiler and it thinks we're executing from SRAM.